example/user/user.override.go

```diff
@@ -9,6 +9,6 @@ import (
 // us to customize conversion behavior. In this example, we set the User's Age
 // based on the Birthday, instead of storing it separately in the DB
 func (m *UserORM) AfterToPB(ctx context.Context, user *User) error {
-	user.Age = uint32(time.Now().Sub(m.Birthday).Hours() / 24 / 365)
+	user.Age = uint32(time.Now().Sub(*m.Birthday).Hours() / 24 / 365)
```

Comment:
This will cause compilation error! Please regenerate and add `user.pb.gorm.go` to this PR.

Reply:
Updated generated code.